theseus_gui/src/pages/project/Index.vue

```diff
@@ -29,7 +29,7 @@
         </Categories>
         <hr class="card-divider" />
         <div class="button-group">
-          <Button color="primary" class="instance-button" @click="install">
+          <Button color="primary" class="instance-button" @click="install(data.versions[0].id)">
```

Comment:
this should probably just be `versions[0].id` (likely what is causing the error below)